### PR TITLE
Show how to close the browser

### PR DESCRIPTION
**Pull request by nylen**
I had a bit of a hard time finding out how to call the `.close()` method.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ browser
   }, stack)
   .test.exists('.forum-topiclines')
   .run(function(err) {
+    browser.close();
```
Comment (puzrin) [inline README.md]: IMHO this example is not perfect to recommend:
1. It's more simple to call `.run(true, callback)` in your case. 
2. `.close()` will work too, but designed for standalone calls. For example, in `after(...)` section of tests.

**Comment by nylen** [comment]
I missed the `teardown` parameter too, so I added an extra sentence about it :)

Do you think it would make sense to make `teardown` default to `true`?

**Comment by puzrin** [comment]
> Do you think it would make sense to make teardown default to true?

I expected primary use for testing, and every test pack usually have multiple action chains. `true` by default would be not convenient here.

**Comment by nylen** [comment]
OK.  I added `true` to the example in the readme.  If you want to take it out, let me know and I'll send a PR.

**Comment by puzrin** [comment]
Merged, thanks

**Comment by nylen** [comment]
I meant, given what you said above, you may not want this change: https://github.com/nodeca/navit/commit/3625b2e3ff5c31202cc84aeb198e219c2718269e#diff-04c6e90faac2675aa89e2176d2eec7d8L42

**Comment by puzrin** [comment]
It's ok and logical in that example IMHO. I meaned, i don't like to change default behaviour in API.

**Pull request merged**